ansible/home/ec2-user/configure_webservers.yml

```
@@ -0,0 +1,68 @@
+- name: Todo Web Servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: What is your Trello API Key?
+
+  - name: trello_api_token
+    prompt: What is your Trello API Token?
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python 3.11
+    ansible.builtin.yum:
+      name: python311
+      state: latest
+    become: yes
```

Not necessarily a recommendation, but worth noting that we can combine these if we wish to:
```suggestion
  - name: Install Git & Python
    ansible.builtin.yum:
      name: 
        - git
        - python311
      state: latest
    become: yes
```